Detections/AlsidForAD/UserAccountIssues.yaml

```diff
@@ -0,0 +1,28 @@
+id: fb9e0b51-8867-48d7-86f4-6e76f2176bf8
+name: Alsid user accounts issues
+description: |
+    'Searches for triggered Indicators of Exposures related to user accounts issues'
+severity: Low
+requiredDataConnectors:
+  - connectorId: AlsidForAD
+    dataTypes:
+      - AlsidForADLog_CL
+queryFrequency: 2h
+queryPeriod: 2h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+relevantTechniques:
+  - T1110
+query: |
+  let SeverityTable=datatable(Severity:string,Level:int) [
+  "low", 1,
+  "medium", 2,
+  "high", 3,
+  "critical", 4
+  ];
+  afad_parser
+  | where MessageType == 0 and (Codename == "C-ACCOUNTS-DANG-SID-HISTORY" or Codename == "C-PRE-WIN2000-ACCESS-MEMBERS" or Codename == "C-PASSWORD-DONT-EXPIRE" or Codename == "C-SLEEPING-ACCOUNTS" or Codename == "C-DANG-PRIMGROUPID" or Codename == "C-PASSWORD-NOT-REQUIRED" or Codename == "C-USER-PASSWORD")
```

Comment:
Change to List, same as I had shown in "Alsid Active Directory attacks pathways" detection.